src/js/game_logic.js

```diff
@@ -1,5 +1,8 @@
 const puzzles = window.puzzlesArray
 
+//total number of wrong guesses
+let numOfincorrectanswers = 0
```

Comment:
When we name variables, we typically use camel case. So in this case, it would be numOfIncorrectAnswers, each new word is capitalized. But Perhaps a better variable name would be incorrectGuesses. It's clear and short.